include/mudObjects/exits.hpp

```diff
@@ -121,6 +125,8 @@ class Exit: public MudObject {
     char        classFlags[4]{};  // class allowed flags
     char        raceFlags[4]{};   // race allowed flags
 
+    std::list<std::string> usedBy; // ids of players that have used this exit
```

Comment:
I'd probably look into using a std::set of some sorts.  A hash set... or a tree set... should both perform better than a list, especially for rooms that are visited fairly often and by a bunch of people.